### PR TITLE
Update to section 12.1

### PR DESCRIPTION
**Pull request by nilabja10201992**
I have made few changes to section 12.1 as I felt appropriate, reference - "Add tips for effective online communication to devguide #9"

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA. This is necessary for legal reasons before we can look at your contribution. Please follow these steps to help rectify the issue:

1. If you don't have an account on b.p.o, please [create one](http://bugs.python.org/user?@template=register)
2. Make sure your GitHub username is listed in "Your Details" at b.p.o
3. If you have not already done so, please sign the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/)
4. If you just signed the CLA, please **wait at least a day** and then check "Your Details" on [bugs.python.org](https://bugs.python.org/) to see if your account has been marked as having signed the CLA (the delay is due to a person having to manually check your signed CLA)
5. Reply here saying you have completed the above steps


Thanks again to your contribution and we look forward to looking at it!

**Comment by nilabja10201992** [comment]
I've completed the above steps

File: communication.rst [hunk 1]
```diff
@@ -15,9 +15,9 @@ python-dev_ is the primary mailing list for discussions about Python's
 development. The list is open to the public and is subscribed to by all core
 developers plus many people simply interested in following Python's
 development. Discussion is focused on issues related to Python's development,
-such as how to handle a specific issue, a PEP, etc.::
+such as how to handle a specific issue, a PEP, etc.
 
-   - Ideas about new functionality should **not** start here and instead
+   - Ideas about new functionality should **NOT** start here and instead
```
Comment (berkerpeksag) [inline communication.rst]: Indentation is not needed here. Could you please change it to:

```rst
such as how to handle a specific issue, a PEP, etc.

- Ideas about new functionality should **not** start here and instead [...]
- Technical support questions should also not be asked [...]
```

Comment (berkerpeksag) [inline communication.rst]: About not -> NOT change: IMO making 'not' bold is enough.

Comment (nilabja10201992) [inline communication.rst]: If I highlight the region with grey, the hyperlinks don't work. So is it okay, if the hyperlink doesn't work?

Comment (berkerpeksag) [inline communication.rst]: Grey background is added by GitHub. I was trying to show an example in reST markup. See https://guides.github.com/features/mastering-markdown/#examples

What I was trying to say is that whitespaces in the beginning of the lines can be deleted:

Change

```rst
such as how to handle a specific issue, a PEP, etc.

   - Ideas about new functionality should **not** start here and instead
     should be sent to python-ideas_.
```

to

```rst
such as how to handle a specific issue, a PEP, etc.

- Ideas about new functionality should **not** start here and instead
  should be sent to python-ideas_.
```

Comment (nilabja10201992) [inline communication.rst]: Please review, was I able to do what is meant?

Comment (methane) [inline communication.rst]: LGTM.  Please wait to CLA sign is checked.

**Comment by nilabja10201992** [comment]
Thank you for guiding me

**Comment by nilabja10201992** [comment]
Why isn't my CLA getting signed?

**Comment by willingc** [comment]
@nilabja10201992 Likely one of two reasons. It typically takes at least one business day for the manual approval of the CLA. The other might be that you need to put your GitHub username into your account on bugs.python.org. Please be patient. Folks have put in a lot of extra volunteer time to move the repos over to GitHub this week. Thanks :sunny:

**Comment by nilabja10201992** [comment]
My GitHub username is registered with bugs.python.org and I've signed the CLA on Friday

**Comment by willingc** [comment]
Thanks for the update @nilabja10201992. After someone manually review the signed CLA, you should be all set. 👍 

If you don't see a change in your status by the end of Wednesday, please @ mention me and I'll look into it. I suspect the manual review is having to go through many CLAs at the moment. Thanks in advance for your patience.

**Comment by nilabja10201992** [comment]
Okay, Thank you

**Comment by berkerpeksag** [comment]
Since you already signed the CLA and this is a trivial change, I went ahead and merged this. Thanks!

**Comment by willingc** [comment]
Thanks @berkerpeksag for the merge. Congrats @nilabja10201992 for the contribution.

**Comment by nilabja10201992** [comment]
Thank you for the merge, I'm happy, I was able to contribute

**Pull request merged**